frontend/kesaseteli/youth/public/locales/sv/common.json

```
@@ -101,7 +101,7 @@
       "unlisted_school": "Min skola finns inte på listan",
       "student_in_helsinki_but_not_resident": "Jag bor inte i Helsingfors, men jag går i skola i Helsingfors.",
       "moving_to_helsinki": "Jag bor inte ännu i Helsingfors, men håller på att flytta dit före 1. juni.",
-      "underage_or_overage": "jag hör inte baserat på åldern till Mina målgruppen, en hör i skolan till rätt årsklass.",
+      "underage_or_overage": "Jag hör inte baserat på åldern till målgruppen, en hör i skolan till rätt årsklass.",
```

```suggestion
      "underage_or_overage": "Jag tillhör inte målgruppen åldersmässigt men det gör jag årsklassmässigt.",
```

I would say that this sentence is not Swedish at all! 😄 The changes makes it a bit better anyway.

Event the Google translate gives a better translation: 

`Jag tillhör inte målgruppen åldersmässigt men det gör jag årsklassmässigt.`

But I would say that it's not good either.